app/aqua-ames.hoon

```diff
@@ -0,0 +1,81 @@
+::  This needs a better SDN solution.  Every ship should have an IP
+::  address, and we should eventually test changing those IP
+::  addresses.
+::
+::  For now, we broadcast every packet to every ship and rely on them
+::  to drop them.
+::
+/-  aquarium
+=,  aquarium
+=>  $~  |%
+    +$  move  (pair bone card)
+    +$  card
+      $%  [%poke wire dock %aqua-events (list aqua-event)]
+          [%peer wire dock path]
+          [%pull wire dock ~]
+      ==
+    ::
+    +$  state
+      $:  %0
+          subscribed=_|
+      ==
+    --
+=,  gall
+=|  moves=(list move)
+=|  aqua-event-list=(list aqua-event)
+=|  ships=(list ship)
+|_  $:  bowl
+        state
+    ==
+++  this  .
+++  apex  %_(this moves ~, aqua-event-list ~, ships ~)
+++  abet
+  =?  this  !=(~ aqua-event-list)
+    %-  emit-moves
+    [ost %poke /aqua-events [our %aqua] %aqua-events aqua-event-list]~
+  ::  ~?  !?=(~ moves)  [%aqua-ames-moves (lent moves)]
+  [moves this]
+::
+++  emit-moves
+  |=  ms=(list move)
+  %_(this moves (weld moves ms))
+::
+++  emit-aqua-events
+  |=  aes=(list aqua-event)
+  %_(this aqua-event-list (weld aqua-event-list aes))
+::
+++  poke-aqua-vane-control
+  |=  command=?(%subscribe %unsubscribe)
+  :_  this(subscribed =(command %subscribe))
+  (aqua-vane-control-handler our ost subscribed command)
+::
+++  diff-aqua-effects
```

Comment:
A little comment explaining that this ignores effects other than %restore or %send might be helpful.